src/reducers/pause.js

```diff
@@ -198,22 +252,36 @@ export function getFrames(state: OuterState) {
   return state.pause.frames;
 }
 
-export function getFrameScope(state: OuterState, frameId: ?string) {
+export function getGeneratedFrameScope(state: OuterState, frameId: ?string) {
   if (!frameId) {
     return null;
   }
 
-  return state.pause.frameScopes[frameId];
+  return state.pause.frameScopes.generated[frameId];
 }
 
-export function getSelectedScope(state: OuterState) {
-  const frameId = getSelectedFrameId(state);
-  return getFrameScope(state, frameId);
+export function getFrameScope(
```

Comment:
ha not needed now but `getFrameScopes` would be simpler... its just so hard to different between plural and signular!

Reply:
The type is called `Scope`, so unless we want to rename all of that too, this seems more consistent.